### PR TITLE
Cosmetics

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
the command
```
nix flake show
```
is producing an error in the current master

### CODE REVIEW

**Review by akokoshn — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -633,5 +634,17 @@ callgrind.*
 /*.pub
 /*.crt
 
+/*/*.c
```
[the remark this answers is not in the document]
Comment (AndreyMlashkin) [reply .gitignore:637]: We always had the following lines in .gitignore:

```
# Ignore stuff in the top level dir that shouldn't be checked in
/*.c
/*.cpp
/*.h
...
```
I just want to extend this logic to the sub-folders.

[the remark this answers is not in the document]
Comment (akokoshn) [reply .gitignore:637]: it's make sense only if used different `.gitignore` for root folder and subfoleders.
We are using single one, so let's not ignore the meaningful files.

[the remark this answers is not in the document]
Comment (AndreyMlashkin) [reply .gitignore:637]: Ok, I will remove these lines then

[the remark this answers is not in the document]
Comment (AndreyMlashkin) [reply .gitignore]: done

**Review by akokoshn — APPROVED** [review]

**Pull request merged**